### PR TITLE
Initial Server Changes after TreeEntry.Commit was removed

### PR DESCRIPTION
**Pull request by NicholasCouri**

### CODE REVIEW

**Comment by heliocliu** [comment]
did you do anything unusual with the install/the lockfile?  the changes in there look kind of strange, particularly the parts where some package versions go backwards

**Comment by NicholasCouri** [comment]
Not that I was aware of. I've updated my local npm version and it seems to have fixed it - just released an update for the lock file. 


---
In reply to: [1078747311](https://github.com/microsoft/FluidFramework/pull/9614#issuecomment-1078747311) [](http://example.com/codeflow?ancestors=1078747311,1078747311)

File: server/routerlicious/packages/lambdas/src/scribe/summaryWriter.ts [hunk 1]
```diff
@@ -639,7 +639,7 @@ export class SummaryWriter implements ISummaryWriter {
                     break;
                 }
                 default: {
-                    throw new Error(`Unexpected TreeEntry type: ${treeEntry.type} when converting ITreeEntry.`);
```
Comment (andre4i) [inline server/routerlicious/packages/lambdas/src/scribe/summaryWriter.ts:642]: Is this intentional? I see `treeEntry.type` inspected in the switch statement..

Comment (NicholasCouri) [reply server/routerlicious/packages/lambdas/src/scribe/summaryWriter.ts:642]: yes, I was getting an error as before, there was another type that we were not dealing with (Commit) - now, there are only 3 types, and they are all dealt with on the switch. "Property 'type' does not exist on type 'never'.ts(2339)"

**Review by heliocliu — APPROVED** [review]

**Pull request merged**